src/tic_tac_toe/board.clj

```diff
@@ -72,9 +68,7 @@
   (let [predicate #(= empty-mark %) newboard board] (keep-indexed (fn [i x] (when (predicate x) i))newboard)))
 
 (defn game-won? [board]
-  (if (= 16 (count board)) 
-  (winner? board winning-positions-four-by-four)
-  (winner? board winning-positions)))
+    (winner? board (win-positions board)))
```

Comment:
is there a need for both `game-won?` and `winner?` Can this not just be a single method that is used? It'll be confusing for anyone using the API to decide which one to use.